cargo-dist/src/config/v1/mod.rs

```diff
@@ -481,6 +483,173 @@ impl TomlLayer {
             }
         }
     }
+
+    /// Determines whether the configured install paths are compatible with each other
+    pub fn validate_install_paths(&self) -> DistResult<()> {
```

Comment:
do we want this to return a vec of install paths maybe? (this is something people often want visibility into in the plan step)- this may not be the place for it, but seeing the unit type return made me think of it

Reply:
I don't think it makes sense. If we add a function like `get_install_paths(&self) -> Vec<...>`, we could use `if let Some(paths) = self.get_install_paths()`, but I'm not sure that's worth the effort for this PR.